components/icons/icons.config.js

```diff
@@ -12,6 +13,9 @@ module.exports = {
       height: 200px !important;
       width: 200px !important;
     }
+    p{
```

Comment:
we should not apply css to elements if possible

Reply:
true next time I am in the page I will look into fixing this issue